cfe_internal/enterprise/federation/federation.cf

```diff
@@ -269,6 +276,33 @@ bundle agent transport_user
       "need to fix incorrect ssh context for transport user but restorecon path in $(sys.libdir)/paths.cf ($paths.restorecon) does not resolve";
 }
 
+bundle agent clean_when_off
+# @brief cleanup changes made for federated reporting on a feeder
+# NOTE: a superhub turned off by removing federation-config.json or setting
+# target_state=off will not revert schema changes.
+# federation_manage_files will always run regardless of off or not
+# so as to be prepared for enablement via Mission Portal UI
+{
+  vars:
+    "user" string => "$(cfengine_enterprise_federation:transport_user.user)";
+    "home" string => "$(cfengine_enterprise_federation:transport_user.home)";
+
+  users:
+    "$(user)"
+      policy => "absent";
+
+  files::
+    "$(home)"
+      delete => tidy;
+
+  commands:
+    # _stdlib_path_exists_<command> and paths.<command> are defined is masterfiles/lib/paths.cf
+    selinux_enabled.default:_stdlib_path_exists_semanage.default:_stdlib_path_exists_restorecon::
+      "$(paths.semanage) -d -t ssh_home_t '$(home)/.ssh(/.*)?'";
+      "$(paths.restorecon) -R -F $(home)/.ssh/";
```

Comment:
This is removed in `files:`, right?

Reply:
Yes. It would be. But this would remove the entry from selinux file contexts.

Reply:
That's what the `semanage` does, the `restorecon` is not needed here AFAICT.